{
    "repository": "cenpy-devs/cenpy",
    "stars": 168,
    "language": "python",
    "size": 23308,
    "clone_url": "https://github.com/cenpy-devs/cenpy.git",
    "timestamp": "2023-06-16T11:33:27.684659Z",
    "clone_success": true,
    "number of actions": 0,
    "number_of_test_actions": 1,
    "actions_successful": true,
    "number_of_actions": 1,
    "actions_build_tools": [
        "pytest"
    ],
    "actions_test_build_tools": [
        "pytest"
    ],
    "actions_tests": [
        {
            "classname": "cenpy.tests.test_explorer.TestExplorer",
            "name": "test_available",
            "time": 0.115,
            "results": [
                {
                    "result": "Passed",
                    "message": "",
                    "type": ""
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_explorer.TestExplorer",
            "name": "test_explain",
            "time": 0.117,
            "results": [
                {
                    "result": "Passed",
                    "message": "",
                    "type": ""
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_explorer.TestExplorer",
            "name": "test_fipstable",
            "time": 0.576,
            "results": [
                {
                    "result": "Passed",
                    "message": "",
                    "type": ""
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_functional",
            "name": "test_all",
            "time": 14.75,
            "results": [
                {
                    "result": "Failure",
                    "message": "KeyError: 'Response from API is malformed. You may have submitted too many queries, formatted the request incorrectly, or experienced significant network connectivity issues. Check to make sure that your inputs, like placenames, are spelled correctly, and that your geographies match the level at which you intend to query. The original error from the Census is:\\\\n(API ERROR 400:Unable to complete operation.([]))'",
                    "type": null
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_functional_products",
            "name": "test_all",
            "time": 36.563,
            "results": [
                {
                    "result": "Failure",
                    "message": "TypeError: sjoin() got an unexpected keyword argument 'predicate'",
                    "type": null
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_parser.Geoparser_Test",
            "name": "test_pysal_holed_multi_polygon",
            "time": 8.304,
            "results": [
                {
                    "result": "Failure",
                    "message": "KeyError: 'Response from API is malformed. You may have submitted too many queries, formatted the request incorrectly, or experienced significant network connectivity issues. Check to make sure that your inputs, like placenames, are spelled correctly, and that your geographies match the level at which you intend to query. The original error from the Census is:\\\\n(API ERROR 400:Unable to complete operation.([]))'",
                    "type": null
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_parser.Geoparser_Test",
            "name": "test_pysal_holed_polygon",
            "time": 8.088,
            "results": [
                {
                    "result": "Failure",
                    "message": "KeyError: 'Response from API is malformed. You may have submitted too many queries, formatted the request incorrectly, or experienced significant network connectivity issues. Check to make sure that your inputs, like placenames, are spelled correctly, and that your geographies match the level at which you intend to query. The original error from the Census is:\\\\n(API ERROR 400:Unable to complete operation.([]))'",
                    "type": null
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_parser.Geoparser_Test",
            "name": "test_pysal_multi_polygon",
            "time": 11.85,
            "results": [
                {
                    "result": "Failure",
                    "message": "KeyError: 'Response from API is malformed. You may have submitted too many queries, formatted the request incorrectly, or experienced significant network connectivity issues. Check to make sure that your inputs, like placenames, are spelled correctly, and that your geographies match the level at which you intend to query. The original error from the Census is:\\\\n(API ERROR 400:Unable to complete operation.([]))'",
                    "type": null
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_parser.Geoparser_Test",
            "name": "test_pysal_polygon",
            "time": 8.951,
            "results": [
                {
                    "result": "Failure",
                    "message": "KeyError: 'Response from API is malformed. You may have submitted too many queries, formatted the request incorrectly, or experienced significant network connectivity issues. Check to make sure that your inputs, like placenames, are spelled correctly, and that your geographies match the level at which you intend to query. The original error from the Census is:\\\\n(API ERROR 400:Unable to complete operation.([]))'",
                    "type": null
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_parser.Geoparser_Test",
            "name": "test_shapely_conversion",
            "time": 7.679,
            "results": [
                {
                    "result": "Passed",
                    "message": "",
                    "type": ""
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_remote.test_remote",
            "name": "test_connection",
            "time": 3.133,
            "results": [
                {
                    "result": "Passed",
                    "message": "",
                    "type": ""
                }
            ],
            "stdout": null,
            "stderr": null
        },
        {
            "classname": "cenpy.tests.test_tiger.test_tiger",
            "name": "test_connection",
            "time": 2.898,
            "results": [
                {
                    "result": "Passed",
                    "message": "",
                    "type": ""
                }
            ],
            "stdout": null,
            "stderr": null
        }
    ]
}